.sphinx/_static/furo_colors.css

```diff
@@ -21,11 +21,13 @@ body {
     --color-admonition-title-background--tip: var(--color-background-primary);
     --color-admonition-title-background--important: var(--color-background-primary);
     --color-admonition-title-background--caution: var(--color-background-primary);
+    --color-admonition-title-background--attention: var(--color-background-primary);
     --color-admonition-title--note: #24598F;
     --color-admonition-title--tip: #24598F;
     --color-admonition-title--important: #C7162B;
     --color-admonition-title--caution: #F99B11;
-    --color-highlighted-background: #EbEbEb;
+    --color-admonition-title--attention: #F99B11;
```

Comment:
For consistency, we should add that to the dark colours as well (since the others are defined there as well).